modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -120,12 +120,19 @@ class ImagingUploader extends Component {
         );
```

Comment:
The compiled file above (modules/imaging_uploader/js/index.js) shouldn't be committed to git.

Reply:
Ok, done.